### PR TITLE
Chamaka/master

### PR DESCRIPTION
**Pull request by chamaka1**
website will now have to send startAdjust and mass:
startAdjust = int(data_store["inputs"]["startAdjust"])
m = int(data_store["inputs"]["mass"])

### CODE REVIEW

**Comment by khanguslee** [comment]
Can you explain what this will be used for?

File: power_model_race.py [hunk 1]
```diff
@@ -51,6 +48,10 @@
 trap1 = interps_loaded[3]
 #trap2
 trap2 = interps_loaded[4]
+#startAdjust
+startAdjust = interps_loaded[5]
+#mass
+m = interps_loaded[6]
```
Comment (khanguslee) [inline power_model_race.py]: Just rename `m` to `mass`  then you won't need any of the comments above each variable name. There's no point commenting a variable as "startAdjust" if the variable name is called `startAdjust`. 

A more meaningful comment would be:
``` python
# A calibrated starting position
startAdjust = interps_loaded[5]
```

File: rider_plan_finder.py [hunk 2]
```diff
@@ -53,6 +53,12 @@
 trap2 = int(data_store["inputs"]["EndTrap"]) 
 
 traplength = trap2 - trap1
+
+#start adjust variable for slopef
```
Comment (khanguslee) [inline rider_plan_finder.py]: Comments should have a single space after # (to please the pylint overlords)
```
# start adjust variable for slopef
```

**Comment by khanguslee** [comment]
Also, can you add some trello cards so we know what else to add onto the website?

File: power_model_race.py (hunk 1 above)
Comment (chamaka1) [reply power_model_race.py]: yeah true but m is the variable for mass so ill just leave it

**Comment by chamaka1** [comment]
> Can you explain what this will be used for?

**Pull request merged**